OracleJava/11/Dockerfile

```diff
@@ -28,8 +28,8 @@ FROM oraclelinux:7-slim as builder
 LABEL maintainer="Aurelio Garcia-Ribeyro <aurelio.garciaribeyro@oracle.com>"
 
 RUN yum install -y gzip tar
-		
-	
+        
+    
```

Comment:
If you're cleaning up anyway, please remove the spaces from these lines. There's an option in VS Code (and most editors/IDEs) that automatically remove spaces from the end of lines when saving, which should be enabled.

Reply:
Removed whitespaces